spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -1754,7 +1756,8 @@ public Object resolveArgument(MethodParameter parameter, Message<?> message) {
 		public KafkaListenerErrorHandler consumeMultiMethodException(MultiListenerBean listener) {
 			return (m, e) -> {
 				listener.errorLatch.countDown();
-				return null;
+				String payload = m.getPayload().toString().toUpperCase();
```

Comment:
> Did you change something in the logic I have missed that thrown exception is returned as normal reply even if that was void method?
Plus how that happened that junk was converted into upper case?

Yes i change logic here.
Should i restore the code and write another tests unit?